tmux/demo-ui.yml

```diff
@@ -0,0 +1,91 @@
+
+# Run on project start, after the first time
+# on_project_restart: command
+
+# Run on project exit ( detaching from tmux session )
+# on_project_exit: command
+
+# Run on project stop
+# on_project_stop: command
+
+# Runs in each window and pane before window/pane specific commands. Useful for setting up interpreter versions.
+# pre_window: rbenv shell 2.0.0-p247
+
+# Pass command line options to tmux. Useful for specifying a different tmux.conf.
+# tmux_options: -f ~/.tmux.mac.conf
+tmux_options: -f <%= ENV["ANGEL_WORKSPACE_DIR"] %>/tmux/tmux.conf
+
+windows:
+  - datahub: ros2 run ros_tcp_endpoint default_server_endpoint --ros-args
+      -r __ns:=${ROS_NAMESPACE}
+      -p ROS_IP:=0.0.0.0
+  - image_converter: ros2 run angel_datahub ImageConverter --ros-args
+      -r __ns:=${ROS_NAMESPACE}
+      -p topic_input_images:=PVFramesNV12
+      -p topic_output_images:=PVFramesRGB
+  # Detectors
+  - activity_detector: ros2 run angel_system_nodes uho_activity_detector --ros-args
+      -r __ns:=${ROS_NAMESPACE}
+      -p frames_per_det:=32
+  - detector: ros2 run angel_system_nodes object_detector_with_descriptors --ros-args
+      -r __ns:=${ROS_NAMESPACE}
+      -p detection_threshold:=0.05
+  # Nodes
+  - task_monitor: ros2 run angel_system_nodes task_monitor --ros-args
+      -r __ns:=${ROS_NAMESPACE}
+      -p det_topic:=ActivityDetections
+      -p task_update_topic:=TaskUpdates
+      -p task_steps:=${CONFIG_DIR}/mit_ll_eval_one_coffee_recipe_steps_v2.json
+      -p task_trigger_thresholds:=${CONFIG_DIR}/mit_ll_eval_one_coffee_recipe_thresholds_v2.json
+#  - spatial_mapper: ros2 run angel_system_nodes spatial_mapper --ros-args
+#      -r __ns:=${ROS_NAMESPACE}
+#      -p spatial_map_topic:=SpatialMapData
+#      -p det_topic:=Detections
+#      -p det_3d_topic:=ObjectDetections3d
+#      -p headset_pose_topic:=HeadsetPoseData
+  - feedback_generator: ros2 run angel_system_nodes feedback_generator --ros-args
+      -r __ns:=${ROS_NAMESPACE}
+      -p activity_detector_topic:=ActivityDetections
+      -p object_detection_topic:=ObjectDetections3d
+      -p task_monitor_topic:=TaskUpdates
+      -p arui_update_topic:=AruiUpdates
+  # Display frames with relatively low latency (as opposed to device portal)
+  #- rqt_2d_detections: rqt -s rqt_image_view/ImageView
+  #    --args ${ROS_NAMESPACE}/PVFramesRGB
+  #    --ros-args -p _image_transport:=raw
+  - simple_2d_overlay: ros2 run angel_debug Simple2dDetectionOverlay --ros-args
+      -r __ns:=${ROS_NAMESPACE}
+      -p topic_input_images:=PVFramesRGB
+      -p topic_input_det_2d:=ObjectDetections
+      -p topic_output_images:=pv_image_detections_2d
+  - compress_image: ros2 run image_transport republish raw compressed --ros-args 
+       -r __ns:=${ROS_NAMESPACE}
+       --remap in:=pv_image_detections_2d 
+       --remap out/compressed:=pv_image_detections_2d/compressed
+  # Demo UI
+  - demo_ui: ros2 launch src/angel_utils/demo_ui/launch/websocket.launch
```

Comment:
Could this be told to also operate under the same `ROS_NAMESPACE` as everything else?